website/content/docs/secrets/terraform.mdx

```diff
@@ -0,0 +1,167 @@
+
+    By default, the secrets engine will mount at the name of the engine. To
+    enable the secrets engine at a different path, use the `-path` argument.
+
+2.  Configure Vault to connect and authenticate to Terraform Cloud:
+
+    ```text
+    $ vault write terraform/config \
+        token=Vhz7652ba4c-0f6e-8e75-5724-5e083d72cfe4
+    Success! Data written to: terraform/config
+    ```
+
+    See [Terraform Cloud's documentation on API
+    tokens](https://www.terraform.io/docs/cloud/users-teams-organizations/api-tokens.html)
+    to determine the appropriate API token for use with the secret engine. In
+    order to perform all operations, a User API token is recommended. 
+
+3.  Configure a role that maps a name in Vault to a Terraform Cloud User. At
+    this time the Terraform Cloud API does not allow dynamic user generation. As
+    a result this secret engine creates dynamic API tokens for an existing user,
+    and manages the lifecycle of that API token. You will need to know the User
+    ID in order to generate User API tokens for that user. You can use the
+    Terraform Cloud [Account
+    API](https://www.terraform.io/docs/cloud/api/account.html) to find the
+    desired User ID.
+
+    ```text
+    $ vault write terraform/roles/my-role user_id=user-12345abcde
+    Success! Data written to: terraform/roles/my-role
+    ```
+
+## Usage
+
+After the secrets engine is configured and a user/machine has a Vault token with
+the proper permission, it can generate credentials.
+
+Generate a new credential by reading from the `/creds` endpoint with the name
+of the role:
+
+```shell-session
+$ vault read terraform/creds/my-role
+Key                Value
+---                -----
+lease_id           terraform/creds/my-user/A_LEASE_ID_PdvmJjACTtKrY2I
+lease_duration     180s
+lease_renewable    true
+token              TJFDSIFDSKFEKZX.FKFKA.akjlfdiouajlkdakadfiowe
+token_id           at-123acbdfask
+```
+
+## Organization, Team, and User Roles
+
+Terraform Cloud supports three distinct types of API tokens; Organizations,
+Teams, and Users. Each token type has distinct access levels and generation
+workflows. A given Vault role can manage any one of the three types at a time,
+however there are important differences to be aware of.
+
+### Organization and Team roles 
+
+The Terraform Cloud API limits both Organization and Team roles to **one active
+token at any given time**. Generating a new Organization or Team API token by
+reading the credentials in Vault or otherwise generating them on
+[app.terraform.io](https://app.terraform.io) will effectively revoke **any**
+existing API token for that Organization or Team. 
+
+Due to this behavior, Organization and Team API tokens created by Vault will be
+stored and returned on future requests, until the credentials get rotated. This
+is to prevent unintentional revocation of tokens that are currently in-use.
+
+Below is an example of creating a Vault role to manage manage an Organization
```

Comment:
double manage. For all your management needs.

Reply:
lol fixed in 475b193a1 😆